styles.css

```diff
@@ -0,0 +1,524 @@
+  font-size: 1.5rem;
+}
+
+#navigation .navlinks.open {
+  display: flex;
+  justify-content: start;
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100%;
+  background-color: #1c1a19;
+}
+
+#navigation .navlinks.open .links {
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 2rem;
+  margin-top: 5rem;
+  width: 100%;
+}
+
+#navigation .navlinks.open .links :nth-child(4) {
+  display: none;
+}
+
+#navigation .navlinks.open .links li {
+  border-bottom: 1px solid #6f6c6b;
+  padding: 1rem;
+  width: 100%;
+}
+
+#navigation .navlinks.open .links li a {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 2.75rem;
+}
+
+#navigation .toggle {
+  height: 2.188rem;
+  width: 2.188rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  transition: all 0.5s ease-in-out;
+  margin-right: 1.5rem;
+  cursor: pointer;
+}
+
+#navigation .toggle .menu {
+  position: relative;
+  height: 0.25rem;
+  width: 1.875rem;
+  border-radius: 5px;
+  background: #fff;
+  transition: all 0.5s ease-in-out;
+}
+
+#navigation .toggle .menu::before,
+#navigation .toggle .menu::after {
+  content: "";
+  position: absolute;
+  height: 0.25rem;
+  width: 1.875rem;
+  border-radius: 5px;
+  background: #fff;
+  transition: all 0.5s ease-in-out;
+}
+
+#navigation .toggle .menu::before {
+  transform: translateY(-10px);
+}
+
+#navigation .toggle .menu::after {
+  transform: translateY(10px);
+}
+
+#navigation .toggle.open .menu {
+  transform: translateX(-35px);
+  background-color: transparent;
+}
+
+#navigation .toggle.open .menu::before {
+  transform: rotate(45deg) translate(24px, -24px);
+}
+
+#navigation .toggle.open .menu::after {
+  transform: rotate(-45deg) translate(24px, 24px);
+}
+
+#portfolio {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 1rem;
+}
+
+#portfolio .title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 2.5rem 0;
+}
+
+#portfolio .title h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #1c1a19;
+}
+
+#portfolio .title hr {
+  margin-top: 1rem;
+  background-color: #1c1a19;
+  width: 90%;
+}
+
+#portfolio .cards {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 1.5rem;
+}
+
+#portfolio .cards .placeholder .plchdr_img img {
+  width: 100%;
+  height: 100%;
+}
+
+#portfolio .cards .placeholder .plchdr_text h3 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+#portfolio .cards .placeholder .plchdr_text p {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+#portfolio .cards .placeholder .plchdr_text ul {
+  display: flex;
```

Comment:
- [ ] Please align `.placeholder` class using flex box as stated in the project requirement 

> In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

- [ ] Please add a bit more gap between the elements of the project card to match the [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=bFO3LY97dTXnhgBF-0) as per [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements)

Figma design:

<img width="186" alt="Screenshot 2023-03-24 at 10 51 49 AM" src="https://user-images.githubusercontent.com/44798044/227438191-39b03f1f-c9b5-4b2a-aa91-f2707f70a400.png">

Your deisgn:


<img width="368" alt="Screenshot 2023-03-24 at 10 52 16 AM" src="https://user-images.githubusercontent.com/44798044/227438234-d89cb747-b96d-44fa-b5ff-aacdb0b51e00.png">